backend/analysis-manager/main.py

```diff
@@ -10,8 +20,20 @@ def handler(request: Request) -> typing.ResponseReturnValue:
     :param request: the incoming request
     :return: an empty successful response
     """
-
     # TODO: validate incoming request
+    try:
+        validate_request = AnalyzeTextAreaModel(
```

Comment:
If you use the [`model_validate`](https://docs.pydantic.dev/latest/api/base_model/#pydantic.BaseModel.model_validate) class method, pydantic will automatically parse out everything for you.

Since the form names are in kebab case but Python wants snake case, you'll need to use an [alias generator](https://docs.pydantic.dev/latest/concepts/alias/#aliaspath-and-aliaschoices) in the model config.

Reply:
This indeed seems like a better way of doing this. Thanks!